diff -ru gtkD-2.18/src/gtkc/atktypes.d gtkD-2.18-gdc/src/gtkc/atktypes.d
--- gtkD-2.18/src/gtkc/atktypes.d	2009-10-11 23:00:21.847578720 +0200
+++ gtkD-2.18-gdc/src/gtkc/atktypes.d	2009-10-11 22:59:33.204254281 +0200
@@ -48,6 +48,7 @@
  */
 public enum AtkHyperlinkStateFlags
 {
+	GDC_BUG_WORKAROUND,
 	IS_INLINE = 1 << 0
 }
 /**
@@ -59,6 +60,7 @@
  */
 public enum AtkRole
 {
+	GDC_BUG_WORKAROUND,
 	INVALID = 0,
 	ACCEL_LABEL,
 	ALERT,
@@ -173,6 +175,7 @@
  */
 public enum AtkRelationType
 {
+	GDC_BUG_WORKAROUND,
 	NULL = 0,
 	CONTROLLED_BY,
 	CONTROLLER_FOR,
@@ -269,6 +272,7 @@
  */
 public enum AtkTextAttribute
 {
+	GDC_BUG_WORKAROUND,
 	INVALID = 0,
 	LEFT_MARGIN,
 	RIGHT_MARGIN,
diff -ru gtkD-2.18/src/gtkc/cairotypes.d gtkD-2.18-gdc/src/gtkc/cairotypes.d
--- gtkD-2.18/src/gtkc/cairotypes.d	2009-10-11 23:00:21.470911994 +0200
+++ gtkD-2.18-gdc/src/gtkc/cairotypes.d	2009-10-11 22:59:33.207586269 +0200
@@ -178,6 +178,7 @@
  */
 public enum cairo_text_cluster_flags_t
 {
+	GDC_BUG_WORKAROUND,
 	BACKWARD = 0x00000001
 }
 alias cairo_text_cluster_flags_t CairoTextClusterFlags;
@@ -342,6 +343,7 @@
  */
 public enum cairo_content_t
 {
+	GDC_BUG_WORKAROUND,
 	COLOR = 0x1000,
 	ALPHA = 0x2000,
 	COLOR_ALPHA = 0x3000
@@ -440,6 +442,7 @@
  */
 public enum cairo_status_t
 {
+	GDC_BUG_WORKAROUND,
 	SUCCESS = 0,
 	NO_MEMORY,
 	INVALID_RESTORE,
diff -ru gtkD-2.18/src/gtkc/gdkpixbuftypes.d gtkD-2.18-gdc/src/gtkc/gdkpixbuftypes.d
--- gtkD-2.18/src/gtkc/gdkpixbuftypes.d	2009-10-11 23:00:21.870920235 +0200
+++ gtkD-2.18-gdc/src/gtkc/gdkpixbuftypes.d	2009-10-11 22:59:33.197586115 +0200
@@ -46,6 +46,7 @@
 	GDK_PIXDATA_COLOR_TYPE_RGBA = 0x02,
 	GDK_PIXDATA_COLOR_TYPE_MASK = 0xff,
 	/+* width, support 8bits only currently +/
+	GDC_BUG_WORKAROUND,
 	GDK_PIXDATA_SAMPLE_WIDTH_8 = 0x01 << 16,
 	GDK_PIXDATA_SAMPLE_WIDTH_MASK = 0x0f << 16,
 	/+* encoding +/
@@ -71,6 +72,7 @@
 	GDK_PIXDATA_DUMP_PIXDATA_STRUCT = 1,
 	GDK_PIXDATA_DUMP_MACROS = 2,
 	/+* type of variables to use +/
+	GDC_BUG_WORKAROUND,
 	GDK_PIXDATA_DUMP_GTYPES = 0,
 	GDK_PIXDATA_DUMP_CTYPES = 1 << 8,
 	GDK_PIXDATA_DUMP_STATIC = 1 << 9,
@@ -87,6 +89,7 @@
  */
 public enum GdkPixbufFormatFlags
 {
+	GDC_BUG_WORKAROUND,
 	WRITABLE = 1 << 0,
 	SCALABLE = 1 << 1,
 	THREADSAFE = 1 << 2
diff -ru gtkD-2.18/src/gtkc/gdktypes.d gtkD-2.18-gdc/src/gtkc/gdktypes.d
--- gtkD-2.18/src/gtkc/gdktypes.d	2009-10-11 23:00:22.110912483 +0200
+++ gtkD-2.18-gdc/src/gtkc/gdktypes.d	2009-10-11 22:59:33.197586115 +0200
@@ -116,6 +116,7 @@
  */
 public enum GdkGrabStatus
 {
+	GDC_BUG_WORKAROUND,
 	SUCCESS = 0,
 	ALREADY_GRABBED = 1,
 	INVALID_TIME = 2,
@@ -200,6 +201,7 @@
  */
 public enum GdkGCValuesMask
 {
+	GDC_BUG_WORKAROUND,
 	FOREGROUND = 1 << 0,
 	BACKGROUND = 1 << 1,
 	FONT = 1 << 2,
@@ -310,6 +312,7 @@
  */
 public enum GdkSubwindowMode
 {
+	GDC_BUG_WORKAROUND,
 	CLIP_BY_CHILDREN = 0,
 	INCLUDE_INFERIORS = 1
 }
@@ -482,6 +485,7 @@
  */
 public enum GdkPixbufRotation
 {
+	GDC_BUG_WORKAROUND,
 	ROTATE_NONE = 0,
 	ROTATE_COUNTERCLOCKWISE = 90,
 	ROTATE_UPSIDEDOWN = 180,
@@ -627,6 +631,7 @@
  */
 public enum GdkCursorType
 {
+	GDC_BUG_WORKAROUND,
 	X_CURSOR = 0,
 	ARROW = 2,
 	BASED_ARROW_DOWN = 4,
@@ -785,6 +790,7 @@
  */
 public enum GdkWindowHints
 {
+	GDC_BUG_WORKAROUND,
 	HINT_POS = 1 << 0,
 	HINT_MIN_SIZE = 1 << 1,
 	HINT_MAX_SIZE = 1 << 2,
@@ -826,6 +832,7 @@
  */
 public enum GdkGravity
 {
+	GDC_BUG_WORKAROUND,
 	NORTH_WEST = 1,
 	NORTH,
 	NORTH_EAST,
@@ -933,6 +940,7 @@
  */
 public enum GdkWindowAttributesType
 {
+	GDC_BUG_WORKAROUND,
 	WA_TITLE = 1 << 1,
 	WA_X = 1 << 2,
 	WA_Y = 1 << 3,
@@ -977,6 +985,7 @@
  */
 public enum GdkModifierType
 {
+	GDC_BUG_WORKAROUND,
 	SHIFT_MASK = 1 << 0,
 	LOCK_MASK = 1 << 1,
 	CONTROL_MASK = 1 << 2,
@@ -1022,6 +1031,7 @@
  */
 public enum GdkWMDecoration
 {
+	GDC_BUG_WORKAROUND,
 	DECOR_ALL = 1 << 0,
 	DECOR_BORDER = 1 << 1,
 	DECOR_RESIZEH = 1 << 2,
@@ -1052,6 +1062,7 @@
  */
 public enum GdkWMFunction
 {
+	GDC_BUG_WORKAROUND,
 	FUNC_ALL = 1 << 0,
 	FUNC_RESIZE = 1 << 1,
 	FUNC_MOVE = 1 << 2,
@@ -1154,6 +1165,7 @@
  */
 public enum GdkEventType
 {
+	GDC_BUG_WORKAROUND,
 	NOTHING = -1,
 	DELETE = 0,
 	DESTROY = 1,
@@ -1252,6 +1264,7 @@
  */
 public enum GdkEventMask
 {
+	GDC_BUG_WORKAROUND,
 	EXPOSURE_MASK = 1 << 1,
 	POINTER_MOTION_MASK = 1 << 2,
 	POINTER_MOTION_HINT_MASK = 1 << 3,
@@ -1349,6 +1362,7 @@
  */
 public enum GdkNotifyType
 {
+	GDC_BUG_WORKAROUND,
 	ANCESTOR = 0,
 	VIRTUAL = 1,
 	INFERIOR = 2,
@@ -1391,6 +1405,7 @@
  */
 public enum GdkWindowState
 {
+	GDC_BUG_WORKAROUND,
 	WITHDRAWN = 1 << 0,
 	ICONIFIED = 1 << 1,
 	MAXIMIZED = 1 << 2,
@@ -1470,6 +1485,7 @@
  */
 public enum GdkDragAction
 {
+	GDC_BUG_WORKAROUND,
 	ACTION_DEFAULT = 1 << 0,
 	ACTION_COPY = 1 << 1,
 	ACTION_MOVE = 1 << 2,
@@ -1497,6 +1513,7 @@
  */
 public enum GdkInputCondition
 {
+	GDC_BUG_WORKAROUND,
 	READ = 1 << 0,
 	WRITE = 1 << 1,
 	EXCEPTION = 1 << 2
diff -ru gtkD-2.18/src/gtkc/giotypes.d gtkD-2.18-gdc/src/gtkc/giotypes.d
--- gtkD-2.18/src/gtkc/giotypes.d	2009-10-11 23:00:21.384245390 +0200
+++ gtkD-2.18-gdc/src/gtkc/giotypes.d	2009-10-11 22:59:33.200920268 +0200
@@ -32,6 +32,7 @@
  */
 public enum GFileQueryInfoFlags
 {
+	GDC_BUG_WORKAROUND,
 	NONE = 0,
 	NOFOLLOW_SYMLINKS = (1 << 0) /+*< nick=nofollow-symlinks >+/
 }
@@ -43,6 +44,7 @@
  */
 public enum GFileCreateFlags
 {
+	GDC_BUG_WORKAROUND,
 	NONE = 0,
 	PRIVATE = (1 << 0),
 	REPLACE_DESTINATION = (1 << 1)
@@ -55,6 +57,7 @@
  */
 public enum GFileCopyFlags
 {
+	GDC_BUG_WORKAROUND,
 	NONE = 0, /+*< nick=none >+/
 	OVERWRITE = (1 << 0),
 	BACKUP = (1 << 1),
@@ -71,6 +74,7 @@
  */
 public enum GFileMonitorFlags
 {
+	GDC_BUG_WORKAROUND,
 	NONE = 0,
 	WATCH_MOUNTS = (1 << 0)
 }
@@ -84,6 +88,7 @@
  */
 public enum GFilesystemPreviewType
 {
+	GDC_BUG_WORKAROUND,
 	TYPE_IF_ALWAYS = 0,
 	TYPE_IF_LOCAL,
 	TYPE_NEVER
@@ -96,6 +101,7 @@
  */
 public enum GFileAttributeType
 {
+	GDC_BUG_WORKAROUND,
 	TYPE_INVALID = 0,
 	TYPE_STRING,
 	TYPE_BYTE_STRING, /+* zero terminated string of non-zero bytes +/
@@ -115,6 +121,7 @@
  */
 public enum GFileAttributeInfoFlags
 {
+	GDC_BUG_WORKAROUND,
 	NONE = 0,
 	COPY_WITH_FILE = (1 << 0),
 	COPY_WHEN_MOVED = (1 << 1)
@@ -127,6 +134,7 @@
  */
 public enum GFileAttributeStatus
 {
+	GDC_BUG_WORKAROUND,
 	UNSET = 0,
 	SET,
 	ERROR_SETTING
@@ -139,6 +147,7 @@
  */
 public enum GFileType
 {
+	GDC_BUG_WORKAROUND,
 	TYPE_UNKNOWN = 0,
 	TYPE_REGULAR,
 	TYPE_DIRECTORY,
@@ -200,6 +209,7 @@
  */
 public enum GAskPasswordFlags
 {
+	GDC_BUG_WORKAROUND,
 	NEED_PASSWORD = (1 << 0),
 	NEED_USERNAME = (1 << 1),
 	NEED_DOMAIN = (1 << 2),
@@ -257,6 +267,7 @@
  */
 public enum GOutputStreamSpliceFlags
 {
+	GDC_BUG_WORKAROUND,
 	NONE = 0,
 	CLOSE_SOURCE = (1 << 0),
 	CLOSE_TARGET = (1 << 1)
@@ -295,6 +306,7 @@
  */
 public enum GAppInfoCreateFlags
 {
+	GDC_BUG_WORKAROUND,
 	NONE = 0, /+*< nick=none >+/
 	NEEDS_TERMINAL = (1 << 0), /+*< nick=needs-terminal >+/
 	SUPPORTS_URIS = (1 << 1) /+*< nick=supports-uris >+/
@@ -307,6 +319,7 @@
  */
 public enum GMountMountFlags
 {
+	GDC_BUG_WORKAROUND,
 	NONE = 0
 }
 alias GMountMountFlags MountMountFlags;
@@ -317,6 +330,7 @@
  */
 public enum GMountUnmountFlags
 {
+	GDC_BUG_WORKAROUND,
 	NONE = 0,
 	FORCE = (1 << 0)
 }
@@ -328,6 +342,7 @@
  */
 public enum GDriveStartFlags
 {
+	GDC_BUG_WORKAROUND,
 	NONE = 0
 }
 alias GDriveStartFlags DriveStartFlags;
@@ -385,6 +400,7 @@
  */
 public enum GSocketProtocol
 {
+	GDC_BUG_WORKAROUND,
 	UNKNOWN = -1,
 	DEFAULT = 0,
 	TCP = 6,
diff -ru gtkD-2.18/src/gtkc/glibtypes.d gtkD-2.18-gdc/src/gtkc/glibtypes.d
--- gtkD-2.18/src/gtkc/glibtypes.d	2009-10-11 23:00:20.767578950 +0200
+++ gtkD-2.18-gdc/src/gtkc/glibtypes.d	2009-10-11 22:59:33.204254281 +0200
@@ -148,6 +148,7 @@
 
 enum GPriority
 {
+	GDC_BUG_WORKAROUND,
 	HIGH = -100,
 	DEFAULT = 0,
 	HIGH_IDLE = 100,
@@ -163,6 +164,7 @@
  */
 public enum GModuleFlags
 {
+	GDC_BUG_WORKAROUND,
 	BIND_LAZY = 1 << 0,
 	BIND_LOCAL = 1 << 1,
 	BIND_MASK = 0x03
@@ -278,6 +280,7 @@
  */
 public enum GIOFlags
 {
+	GDC_BUG_WORKAROUND,
 	APPEND = 1 << 0,
 	NONBLOCK = 1 << 1,
 	IS_READABLE = 1 << 2, /+* Read only flag +/
@@ -327,6 +330,7 @@
 	G_LOG_FLAG_RECURSION = 1 << 0,
 	G_LOG_FLAG_FATAL = 1 << 1,
 	/+* GLib log levels +/
+	GDC_BUG_WORKAROUND,
 	G_LOG_LEVEL_ERROR = 1 << 2, /+* always fatal +/
 	G_LOG_LEVEL_CRITICAL = 1 << 3,
 	G_LOG_LEVEL_WARNING = 1 << 4,
@@ -600,6 +604,7 @@
  */
 public enum GUnicodeScript
 {
+	GDC_BUG_WORKAROUND,
 	INVALID_CODE = -1,
 	COMMON = 0, /+* Zyyy +/
 	INHERITED, /+* Qaai +/
@@ -734,6 +739,7 @@
  */
 public enum GDateDMY
 {
+	GDC_BUG_WORKAROUND,
 	DAY = 0,
 	MONTH = 1,
 	YEAR = 2
@@ -772,6 +778,7 @@
  */
 public enum GDateMonth
 {
+	GDC_BUG_WORKAROUND,
 	BAD_MONTH = 0,
 	JANUARY = 1,
 	FEBRUARY = 2,
@@ -810,6 +817,7 @@
  */
 public enum GDateWeekday
 {
+	GDC_BUG_WORKAROUND,
 	BAD_WEEKDAY = 0,
 	MONDAY = 1,
 	TUESDAY = 2,
@@ -857,6 +865,7 @@
  */
 public enum GTokenType
 {
+	GDC_BUG_WORKAROUND,
 	EOF = 0,
 	LEFT_PAREN = '(',
 	RIGHT_PAREN = ')',
@@ -994,6 +1003,7 @@
  */
 public enum GSpawnFlags
 {
+	GDC_BUG_WORKAROUND,
 	LEAVE_DESCRIPTORS_OPEN = 1 << 0,
 	DO_NOT_REAP_CHILD = 1 << 1,
 	/+* look for argv[0] inn the path i.e. use execvp() +/
@@ -1058,6 +1068,7 @@
  */
 public enum GFileTest
 {
+	GDC_BUG_WORKAROUND,
 	IS_REGULAR = 1 << 0,
 	IS_SYMLINK = 1 << 1,
 	IS_DIR = 1 << 2,
@@ -1125,6 +1136,7 @@
  */
 public enum GOptionFlags
 {
+	GDC_BUG_WORKAROUND,
 	HIDDEN = 1 << 0,
 	IN_MAIN = 1 << 1,
 	REVERSE = 1 << 2,
@@ -1345,6 +1357,7 @@
  */
 public enum GRegexCompileFlags
 {
+	GDC_BUG_WORKAROUND,
 	CASELESS = 1 << 0,
 	MULTILINE = 1 << 1,
 	DOTALL = 1 << 2,
@@ -1409,6 +1422,7 @@
  */
 public enum GRegexMatchFlags
 {
+	GDC_BUG_WORKAROUND,
 	ANCHORED = 1 << 4,
 	NOTBOL = 1 << 7,
 	NOTEOL = 1 << 8,
@@ -1462,6 +1476,7 @@
  */
 public enum GMarkupParseFlags
 {
+	GDC_BUG_WORKAROUND,
 	DO_NOT_USE_THIS_UNSUPPORTED_FLAG = 1 << 0,
 	TREAT_CDATA_AS_TEXT = 1 << 1,
 	PREFIX_ERROR_POSITION = 1 << 2
@@ -1523,6 +1538,7 @@
  */
 public enum GKeyFileFlags
 {
+	GDC_BUG_WORKAROUND,
 	NONE = 0,
 	KEEP_COMMENTS = 1 << 0,
 	KEEP_TRANSLATIONS = 1 << 1
@@ -1586,6 +1602,7 @@
  */
 public enum GTraverseFlags
 {
+	GDC_BUG_WORKAROUND,
 	LEAVES = 1 << 0,
 	NON_LEAVES = 1 << 1,
 	ALL = LEAVES | NON_LEAVES,
diff -ru gtkD-2.18/src/gtkc/gobjecttypes.d gtkD-2.18-gdc/src/gtkc/gobjecttypes.d
--- gtkD-2.18/src/gtkc/gobjecttypes.d	2009-10-11 23:00:21.017578679 +0200
+++ gtkD-2.18-gdc/src/gtkc/gobjecttypes.d	2009-10-11 22:59:33.197586115 +0200
@@ -32,6 +32,7 @@
 // G_TYPE_*
 enum GType : size_t
 {
+	GDC_BUG_WORKAROUND,
 	INVALID = 0<<2,
 	NONE = 1<<2,
 	INTERFACE = 2<<2,
@@ -89,6 +90,7 @@
  */
 public enum GTypeDebugFlags
 {
+	GDC_BUG_WORKAROUND,
 	NONE = 0,
 	OBJECTS = 1 << 0,
 	SIGNALS = 1 << 1,
@@ -101,6 +103,7 @@
  */
 public enum GTypeFlags
 {
+	GDC_BUG_WORKAROUND,
 	ABSTRACT = (1 << 4),
 	VALUE_ABSTRACT = (1 << 5)
 }
@@ -112,6 +115,7 @@
  */
 public enum GTypeFundamentalFlags
 {
+	GDC_BUG_WORKAROUND,
 	FLAG_CLASSED = (1 << 0),
 	FLAG_INSTANTIATABLE = (1 << 1),
 	FLAG_DERIVABLE = (1 << 2),
@@ -126,6 +130,7 @@
  */
 public enum GParamFlags
 {
+	GDC_BUG_WORKAROUND,
 	READABLE = 1 << 0,
 	WRITABLE = 1 << 1,
 	CONSTRUCT = 1 << 2,
@@ -146,6 +151,7 @@
  */
 public enum GSignalFlags
 {
+	GDC_BUG_WORKAROUND,
 	RUN_FIRST = 1 << 0,
 	RUN_LAST = 1 << 1,
 	RUN_CLEANUP = 1 << 2,
@@ -164,6 +170,7 @@
  */
 public enum GSignalMatchType
 {
+	GDC_BUG_WORKAROUND,
 	ID = 1 << 0,
 	DETAIL = 1 << 1,
 	CLOSURE = 1 << 2,
@@ -180,6 +187,7 @@
  */
 public enum GConnectFlags
 {
+	GDC_BUG_WORKAROUND,
 	AFTER = 1 << 0,
 	SWAPPED = 1 << 1
 }
diff -ru gtkD-2.18/src/gtkc/gtktypes.d gtkD-2.18-gdc/src/gtkc/gtktypes.d
--- gtkD-2.18/src/gtkc/gtktypes.d	2009-10-11 23:00:23.407578179 +0200
+++ gtkD-2.18-gdc/src/gtkc/gtktypes.d	2009-10-11 22:59:33.200920268 +0200
@@ -121,6 +121,7 @@
  */
 public enum GtkDestDefaults
 {
+	GDC_BUG_WORKAROUND,
 	MOTION = 1 << 0, /+* respond to "dragMotion" +/
 	HIGHLIGHT = 1 << 1, /+* auto-highlight +/
 	DROP = 1 << 2, /+* respond to "dragDrop" +/
@@ -135,6 +136,7 @@
  */
 public enum GtkTargetFlags
 {
+	GDC_BUG_WORKAROUND,
 	SAME_APP = 1 << 0, /+*< nick=same-app >+/
 	SAME_WIDGET = 1 << 1, /+*< nick=same-widget >+/
 	OTHER_APP = 1 << 2, /+*< nick=other-app >+/
@@ -148,6 +150,7 @@
  */
 public enum GtkIconLookupFlags
 {
+	GDC_BUG_WORKAROUND,
 	NO_SVG = 1 << 0,
 	FORCE_SVG = 1 << 1,
 	USE_BUILTIN = 1 << 2,
@@ -194,6 +197,7 @@
  */
 public enum GtkRcFlags
 {
+	GDC_BUG_WORKAROUND,
 	FG = 1 << 0,
 	BG = 1 << 1,
 	TEXT = 1 << 2,
@@ -209,6 +213,7 @@
  */
 public enum GtkRcTokenType
 {
+	GDC_BUG_WORKAROUND,
 	INVALID = TokenType.LAST,
 	INCLUDE,
 	NORMAL,
@@ -254,6 +259,7 @@
 
 public enum GtkAccelFlags
 {
+	GDC_BUG_WORKAROUND,
 	VISIBLE = 1 << 0, /+* display inn GtkAccelLabel? +/
 	LOCKED = 1 << 1, /+* is it removable? +/
 	MASK = 0x07
@@ -335,6 +341,7 @@
  */
 public enum GtkAttachOptions
 {
+	GDC_BUG_WORKAROUND,
 	EXPAND = 1 << 0,
 	SHRINK = 1 << 1,
 	FILL = 1 << 2
@@ -560,6 +567,7 @@
 
 public enum GtkPathPriorityType
 {
+	GDC_BUG_WORKAROUND,
 	LOWEST = 0,
 	GTK = 4,
 	APPLICATION = 8,
@@ -999,6 +1007,7 @@
  */
 public enum GtkSignalRunType
 {
+	GDC_BUG_WORKAROUND,
 	RUN_FIRST = SignalFlags.RUN_FIRST,
 	RUN_LAST = SignalFlags.RUN_LAST,
 	RUN_BOTH = (RUN_FIRST | RUN_LAST),
@@ -1015,6 +1024,7 @@
  */
 public enum GtkDialogFlags
 {
+	GDC_BUG_WORKAROUND,
 	MODAL = 1 << 0, /+* call windowSetModal (win, TRUE) +/
 	DESTROY_WITH_PARENT = 1 << 1, /+* call windowSetDestroyWithParent () +/
 	NO_SEPARATOR = 1 << 2 /+* no separator bar above buttons +/
@@ -1033,6 +1043,7 @@
 	/+* GTK returns this if a response widget has no responseId,
 	 * or if the dialog gets programmatically hidden or destroyed.
 	+/
+	GDC_BUG_WORKAROUND,
 	GTK_RESPONSE_NONE = -1,
 	/+* GTK won't return these unless you pass them inn
 	 * as the response for an action widget. They are
@@ -1259,6 +1270,7 @@
 
 public enum GtkTextSearchFlags
 {
+	GDC_BUG_WORKAROUND,
 	VISIBLE_ONLY = 1 << 0,
 	TEXT_ONLY = 1 << 1
 	/+* Possible future plans: SEARCH_CASE_INSENSITIVE, SEARCH_REGEXP +/
@@ -1267,6 +1279,7 @@
 
 public enum GtkTextBufferTargetInfo
 {
+	GDC_BUG_WORKAROUND,
 	BUFFER_CONTENTS = - 1,
 	RICH_TEXT = - 2,
 	TEXT = - 3
@@ -1314,6 +1327,7 @@
  */
 public enum GtkTreeModelFlags
 {
+	GDC_BUG_WORKAROUND,
 	ITERS_PERSIST = 1 << 0,
 	LIST_ONLY = 1 << 1
 }
@@ -1424,6 +1438,7 @@
  */
 public enum GtkCellRendererState
 {
+	GDC_BUG_WORKAROUND,
 	SELECTED = 1 << 0,
 	PRELIT = 1 << 1,
 	INSENSITIVE = 1 << 2,
@@ -1542,6 +1557,7 @@
  */
 public enum GtkUIManagerItemType
 {
+	GDC_BUG_WORKAROUND,
 	AUTO = 0,
 	MENUBAR = 1 << 0,
 	MENU = 1 << 1,
@@ -1600,6 +1616,7 @@
  */
 public enum GtkFileFilterFlags
 {
+	GDC_BUG_WORKAROUND,
 	FILENAME = 1 << 0,
 	URI = 1 << 1,
 	DISPLAY_NAME = 1 << 2,
@@ -1780,6 +1797,7 @@
  */
 public enum GtkPrintCapabilities
 {
+	GDC_BUG_WORKAROUND,
 	CAPABILITY_PAGE_SET = 1 << 0,
 	CAPABILITY_COPIES = 1 << 1,
 	CAPABILITY_COLLATE = 1 << 2,
@@ -1799,6 +1817,7 @@
  */
 public enum GtkCalendarDisplayOptions
 {
+	GDC_BUG_WORKAROUND,
 	SHOW_HEADING = 1 << 0,
 	SHOW_DAY_NAMES = 1 << 1,
 	NO_MONTH_CHANGE = 1 << 2,
@@ -1851,6 +1870,7 @@
  */
 public enum GtkObjectFlags
 {
+	GDC_BUG_WORKAROUND,
 	IN_DESTRUCTION = 1 << 0, /+* Used internally during dispose +/
 	FLOATING = 1 << 1,
 	RESERVED_1 = 1 << 2,
@@ -1879,6 +1899,7 @@
  */
 public enum GtkArgFlags
 {
+	GDC_BUG_WORKAROUND,
 	READABLE = ParamFlags.READABLE,
 	WRITABLE = ParamFlags.WRITABLE,
 	CONSTRUCT = ParamFlags.CONSTRUCT,
@@ -1907,6 +1928,7 @@
  */
 public enum GtkWidgetFlags
 {
+	GDC_BUG_WORKAROUND,
 	TOPLEVEL = 1 << 4,
 	NO_WINDOW = 1 << 5,
 	REALIZED = 1 << 6,
@@ -1991,6 +2013,7 @@
  */
 public enum GtkRecentSortType
 {
+	GDC_BUG_WORKAROUND,
 	NONE = 0,
 	MRU,
 	LRU,
@@ -2008,6 +2031,7 @@
  */
 public enum GtkRecentFilterFlags
 {
+	GDC_BUG_WORKAROUND,
 	URI = 1 << 0,
 	DISPLAY_NAME = 1 << 1,
 	MIME_TYPE = 1 << 2,
diff -ru gtkD-2.18/src/gtkc/Loader.d gtkD-2.18-gdc/src/gtkc/Loader.d
--- gtkD-2.18/src/gtkc/Loader.d	2009-10-11 23:00:20.207173539 +0200
+++ gtkD-2.18-gdc/src/gtkc/Loader.d	2009-10-11 22:59:33.197586115 +0200
@@ -266,6 +266,7 @@
 
 	enum RTLD
 	{
+	GDC_BUG_WORKAROUND,
 		LAZY     = 0x00001,  // Lazy function call binding
 		NOW      = 0x00002,  // Immediate function call binding
 		NOLOAD   = 0x00004,  // No object load
diff -ru gtkD-2.18/src/gtkc/pangotypes.d gtkD-2.18-gdc/src/gtkc/pangotypes.d
--- gtkD-2.18/src/gtkc/pangotypes.d	2009-10-11 23:00:21.780912287 +0200
+++ gtkD-2.18-gdc/src/gtkc/pangotypes.d	2009-10-11 22:59:33.204254281 +0200
@@ -165,6 +165,7 @@
  */
 public enum PangoWeight
 {
+	GDC_BUG_WORKAROUND,
 	THIN = 100,
 	ULTRALIGHT = 200,
 	LIGHT = 300,
@@ -243,6 +244,7 @@
  */
 public enum PangoFontMask
 {
+	GDC_BUG_WORKAROUND,
 	FAMILY = 1 << 0,
 	STYLE = 1 << 1,
 	VARIANT = 1 << 2,
@@ -433,6 +435,7 @@
  */
 public enum PangoScript
 {
+	GDC_BUG_WORKAROUND,
 	INVALID_CODE = -1,
 	COMMON = 0, /+* Zyyy +/
 	INHERITED, /+* Qaai +/
